build-test.xml

```diff
@@ -14370,6 +14374,14 @@ url=${short.hostname}:8080</echo>
 		/>
 	</target>
 
+	<target name="update-dependency-injector">
+		<replace
+			file="${project.dir}/${gradle.file.path}/service.xml"
+			token="dependency-injector=&quot;spring&quot;"
```

Comment:
@kyle-miho When you use spring as dependecy injector, you won't find "dependency-injector=spring" in your service.xml, as we added this property "dependency-injector" when we introduced ds dependency.